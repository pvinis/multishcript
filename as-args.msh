#!/usr/bin/env msh


___bash
ls
___ >mid

___bash mid
echo "below should be the output of the first script!"
echo $1
___

___js mid
console.log("below should be the output of the first script as well!"
console.log(process.argv[1])
___
