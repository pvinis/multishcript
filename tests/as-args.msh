#!/usr/bin/env msh


___bash
ls
___ >mid

___bash mid
echo "below should be the output of the first script!"
echo $1
___


# anything that returns, lives in an envvar, that the next can read

# have a json return obj, maybe keep the output, or the time it took, etc


___js mid
console.log("below should be the output of the first script as well!"
console.log(process.argv[1])
___





/temp/sc1 mid

#!/


echo "below should be the output of the first script!"
echo $1
