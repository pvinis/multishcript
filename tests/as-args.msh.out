below should be the output of the first script!
something.md wow.txt
below should be the output of the first script as well!
something.md wow.txt
